pkg/scan/list_kind.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package scan
+
+import (
+	"fmt"
+	"runtime"
+	"sort"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pivotal/go-ape/pkg/furl"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+func ListKind(res string, baseDir string) ([]string, error) {
```

Comment:
Where is this used, apart from in unit tests?

Reply:
At the moment it is a convenience method for the unit test, also maintains parity with ListImages.

Reply:
Ok. Might be worth a comment to that effect.